hack/ci/e2e-conformance-gcp-cloud-init.yaml

```diff
@@ -0,0 +1,103 @@
+#cloud-config
+hostname: openstack
+password: ubuntu
+chpasswd: { expire: False }
+ssh_pwauth: True
+write_files:
+  - content: |
+      net.ipv4.ip_forward=1
+      net.ipv4.conf.default.rp_filter=0
+      net.ipv4.conf.all.rp_filter=0
+    path: /etc/sysctl.d/devstack.conf
+  - content: |
+      #!/bin/bash
+
+      set -o errexit -o nounset -o pipefail
+
+      # Install kvm / ensure nested virtualization
+      sudo apt-get update && sudo apt-get install qemu-kvm jq net-tools -y
+      kvm-ok
+      sudo modprobe kvm-intel
+
+      # from https://raw.githubusercontent.com/openstack/octavia/master/devstack/contrib/new-octavia-devstack.sh
+      git clone -b stable/victoria https://github.com/openstack/devstack.git /tmp/devstack
+
+      cat <<EOF > /tmp/devstack/localrc
+      GIT_BASE=https://github.com
+
+      # Neutron
+      enable_plugin neutron https://github.com/openstack/neutron stable/victoria
+
+      # Octavia
+      enable_plugin octavia https://github.com/openstack/octavia stable/victoria
+      enable_plugin octavia-dashboard https://github.com/openstack/octavia-dashboard stable/victoria
+      #LIBS_FROM_GIT+=python-octaviaclient
+
+      # Cinder
+      enable_plugin cinderlib https://github.com/openstack/cinderlib stable/victoria
+
+      KEYSTONE_TOKEN_FORMAT=fernet
+
+      SERVICE_TIMEOUT=240
+
+      DATABASE_PASSWORD=secretdatabase
+      RABBIT_PASSWORD=secretrabbit
+      ADMIN_PASSWORD=secretadmin
+      SERVICE_PASSWORD=secretservice
+      SERVICE_TOKEN=111222333444
+      # Enable Logging
+      LOGFILE=/opt/stack/logs/stack.sh.log
+      VERBOSE=True
+      LOG_COLOR=True
+
+      # Pre-requisite
+      ENABLED_SERVICES=key,rabbit,mysql
+      # Nova
+      ENABLED_SERVICES+=,n-api,n-obj,n-cpu,n-cond,n-sch,n-novnc,n-api-meta
+      # Placement service needed for Nova
+      ENABLED_SERVICES+=,placement-api,placement-client
+      # Glance
+      ENABLED_SERVICES+=,g-api,g-reg
+
+      # Octavia-Neutron
+      ENABLED_SERVICES+=,neutron-api,neutron-agent,neutron-dhcp,neutron-l3
+      ENABLED_SERVICES+=,neutron-metadata-agent,neutron-qos
+      # Octavia
+      ENABLED_SERVICES+=,octavia,o-api,o-cw,o-hm,o-hk,o-da
+
+      # Horizon (enable for manual tests)
+      #ENABLED_SERVICES+=,horizon
```

Comment:
not sure we want horizon at all

Reply:
That's why I commented it out (so it is not installed) Just thought it's easier when somebody needs to test it. E.g. I ran those tests against my private GCP account and deployed it with horizon a few times.

Reply:
+1, we shouldn't need it for normal CI runs, but having it easy to find for anyone running manual tests could be helpful.

Reply:
sure, I think we are good and maybe if really need ,we can open it later on